src/external/ACSAdapter/AMSFileManager.ts

```diff
@@ -60,14 +70,26 @@ class AMSFileManager {
     }
 
     public getFileIds(metadata?: Record<string, string>): string[] | undefined {
-        if (!metadata || !metadata.amsReferences) {
+        if (!metadata) {
+            return;
+        }
+
+        if (!metadata.amsReferences && !metadata.amsreferences) {

```

Comment:
is this correct? double same validation?

Reply:
Yes. If there's an attachment, it should have either `amsreferences` or `amsReferences`. We can then short-circuit it as soon as both properties are null.